recipes/taxonid/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "taxonid" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9167b95a28d1dc9ff593d8da497cbdb1833fa81ec13b9a5030f699f8ef045145
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - biopython
+    - brs_utils
+    - colored
+
+test:
+  source_files:
+    - tests
+  imports:
+    - {{ name|lower }}
```

Comment:
Please just use the package name instead of `{{ name|lower }}`, here in the test section and below in the `about` section.

Reply:
ok done, thank you